test/spec/ol/source/wmtssource.test.js

```diff
@@ -157,7 +157,7 @@ describe('ol.source.WMTS', function() {
       });
     });
 
-    it('can create KVP options from spec/ol/format/wmts/arcgis.xml',
+    it('can create REST options from spec/ol/format/wmts/arcgis.xml',
```

Comment:
While you're at it, you could split this test into two - one for KVP (with `requestEncoding` set in the read options) and one for REST.